src/databricks/labs/dqx/pii/config.py

```diff
@@ -0,0 +1,55 @@
+from enum import Enum
```

Comment:
having the same module under one project may be confusing even though it leaves in different folder. Maybe rename it to `pii_config`?